product_pricelist_report_qweb/wizard/product_pricelist.py

```diff
@@ -19,62 +19,60 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ###############################################################################
-from openerp.osv import fields, osv
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
 
 
-class ProductPriceList(osv.osv_memory):
+class ProductPriceList(models.TransientModel):
     _inherit = 'product.price_list'
     _description = 'Price List'
     _rec_name = 'price_list'
 
-    _columns = {
-        'report_format': fields.selection([
-            ('pdf', 'PDF'),
-            # TODO: enable print on controller to HTML
-            # ('html', 'HTML'),
-            ('xls', 'Spreadsheet')], 'Report Format', default='xls'),
-        'cost': fields.boolean('Cost'),
-        'margin_cost': fields.boolean('Exp. Marg. Cost (%)'),
-        'margin_sale': fields.boolean('Exp. Marg. Sale (%)'),
-        'only_prod_pricelist': fields.boolean(
-            'Only products in pricelist', help='If you active this field the '
-            'products that are not in pricelist will have in the report the '
-            'price in zero', default=True),
-        'products_with_price': fields.boolean(
-            'Only products with price', help='If this field is active, only '
-            'will to add the products that have price.'
-        )
-    }
+    report_format = fields.Selection([
```

Comment:
`required=True`

Reply:
This is a model from Odoo, so adding new required fields could cause issues, hence resolving.